examples/mcu-board-support/Cargo.toml

```diff
@@ -17,7 +17,7 @@ links = "mcu_board_support" # just so we can pass metadata to the slint build cr
 path = "lib.rs"
 
 [features]
-pico-st7789 = ["slint/unsafe-single-threaded", "rp-pico", "embedded-hal", "cortex-m-rt", "alloc-cortex-m", "fugit", "cortex-m", "cortex-m", "display-interface", "st7789", "defmt", "defmt-rtt",  "slint/defmt", "shared-bus", "slint/libm", "embedded-dma", "embedded-graphics"]
+pico-st7789 = ["slint/unsafe-single-threaded", "rp-pico", "embedded-hal", "cortex-m-rt", "alloc-cortex-m", "fugit", "cortex-m", "cortex-m", "display-interface", "st7789", "defmt", "defmt-rtt",  "slint/defmt", "shared-bus", "slint/libm", "embedded-dma", "embedded-graphics", "euclid/libm"]
```

Comment:
slint/libm hould imply euclid/libm 

This crate shouldn't depend on Euclid. Or what is the reason it does?

Reply:
The XPT2046 uses it, for example `let mut point = Point2D::new(0u32, 0u32);` and `point += euclid::vec2(i16::MAX as u32 - x, y)`. I thought about changing that code as well, but figured it's out of scope for what is primarily an API change. I can do it though if you'd prefer.

Reply:
ah ok, then let's leave it for now